cpp/iedriver/CommandHandlers/ScreenshotCommandHandler.cpp

```diff
@@ -259,11 +259,18 @@ namespace webdriver {
   }
 
   bool ScreenshotCommandHandler::IsSameColour() {
-    COLORREF firstPixelColour = this->image_->GetPixel(0, 0);
+    BYTE* byteptr = (BYTE*)this->image_->GetBits();
+    int pitch = this->image_->GetPitch();
+    int bytePerPixel = this->image_->GetBPP() / 8;
```

Comment:
This expression works only for 32 bit images. In general thats not correct. For example in case of 16 bit image, there will be 3 bytes per pixel, but in your case that would be 2. I suggest you just hardcode 4